migration/bucket_migrator.go

```diff
@@ -15,6 +15,8 @@ type (
 	BucketMigratorKeyFilter func([]byte) (bool, error)
 )
 
+// AFAIK right now bucket migration is not transactional across buckets, any ideas how to improve that?
```

Comment:
BucketMigrator only works on a single bucket, so I am not sure what you mean here exactly.